assets/dev/scss/frontend/_compatibility.scss

```diff
@@ -19,6 +19,7 @@
 	padding: 0;
 }
 
+// To be removed in Elementor 3.26.0.
```

Comment:
We will need this styling in this filing, because of a Core/Pro dependency.

I am not sure if there another way to handle this.

Reply:
@hein-obox Can you elaborate on this Core/Pro dependency please?
In which scenario is it a problem? (Only upgrading Core and not Pro... etc)

Reply:
@louiswol94 
I have created a separated Pro PR: https://github.com/elementor/elementor-pro/pull/4354

Reply:
So the problem is if the user updates Pro and not Core? Then the styles would not be available, right?
If so, I would rather copy the styles to Pro (and clean it out in a alter version).
Cleaning Core now has a bigger impact.

Reply:
Removed.